src/screens/MapPage/components/SidePanel/SidePanel.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import {
+  Box,
+  Button,
+  CheckBox,
+  Image,
+  Text
+} from 'grommet'
+import Logo from 'images/logo.png'
+import RectangleIcon from 'images/rectangle_icon.svg'
+import Map from 'images/map.png'
+import styled from 'styled-components'
+import LocationDrop from '../LocationDrop'
+
+const StyledButton = styled(Button)`
+  background: white;
+  box-shadow: 0px 2.5px 5px gray;
+  padding: 0.5em;
+  width: 10rem;
+`
+
+const StyledText = styled(Text)`
+  font-family: 'Neuton-Light';
+`
+
+export default function SidePanel() {
+  return (
+    <aside>
+      <Box
+        background='sand'
+        border={{ color: 'kelp', side: 'right', size: 'xsmall' }}
+        height={{ min: '100%' }}
+        pad='small'
+        width='15rem'
+      >
+        <Box gap='xsmall' margin={{ bottom: 'small' }} width='10rem'>
+          <Image fit='contain' margin={{ right: 'auto' }} src={Logo} />
+          <StyledText color='kelp' size='xlarge'>
+            Falkland Islands
+          </StyledText>
+          <LocationDrop />
+        </Box>
+
+        <Box
+          border={{ color: 'kelp', side: 'horizontal', size: 'xsmall' }}
+          gap='xsmall'
+          pad={{ vertical: 'small' }}
+        >
+          <Text color='kelp'>Getting started</Text>
+          <Text size='xsmall'>
+            Use the rectangle tool to draw a box around the
+            area of the map you'd like to learn more about
+          </Text>
+          <StyledButton
+            color='kelp'
+            gap='xsmall'
+            hoverIndicator={{ color: 'indiglo' }}
+            icon={<Image src={RectangleIcon} />}
+            label={<Text size='xsmall'>RECTANGLE TOOL</Text>}
```

Comment:
Can the text here be styled with `text-transform: uppercase` rather than written in capitals?

Reply:
Ah, good catch. Will make the change. Out of curiosity, is this a better choice for accessibility reasons?

Reply:
It might change how a screenreader reads the word. I've heard VoiceOver read capitals differently in the past, but I'm not sure how it makes those decisions about pronunciation.